pages/product/product.html

```diff
@@ -0,0 +1,434 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Product</title>
+</head>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css"
+    integrity="sha512-5A8nwdMOWrSz20fDsjczgUidUBR8liPYU+WymTZP1lmY9G6Oc7HlZv156XqnsgNUzTyMefFTcsFH/tnJE/+xBg=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+<link rel="stylesheet" href="https://uibook.netlify.app/css/components.css" />
+<link href="https://unpkg.com/boxicons@2.1.1/css/boxicons.min.css" rel="stylesheet" />
+<link rel="stylesheet" href="/style.css" />
+<link rel="stylesheet" href="/pages/wishlist/wishlist.css">
+<link rel="stylesheet" href="/pages/product/product.css">
+
+<body>
+    <header class="header">
+        <div class="logo">ByBook</div>
+
+        <div class="search-container">
+            <form action="">
+                <input id="search" type="search" placeholder="Search" />
+                <label for="search" class="bx bx-search-alt-2"></label>
+            </form>
+        </div>
+
+        <div class="action-icon">
+            <a href="#" class="login"><i class='bx bxs-user-circle'></i></a>
```

Comment:
You have not added any text over here for link, it shows just a white area
![navbar](https://user-images.githubusercontent.com/90546508/154789645-7ae7330b-ab7f-4afd-9f06-a4e2880e682f.PNG)
.